docs/csharp/programming-guide/classes-and-structs/structs.md

```diff
@@ -7,44 +7,36 @@ helpviewer_keywords:
 ms.assetid: b7cf4ff2-0eb7-4e5c-93d5-b2196b4f5d89
 ---
 # Structs (C# Programming Guide)
-Structs are defined by using the [struct](../../../csharp/language-reference/keywords/struct.md) keyword, for example:  
+
+Structs are defined by using the [struct](../../language-reference/keywords/struct.md) keyword, for example:  
   
- [!code-csharp[csProgGuideObjects#39](../../../csharp/programming-guide/classes-and-structs/codesnippet/CSharp/structs_1.cs)]  
+[!code-csharp[csProgGuideObjects#39](./codesnippet/CSharp/structs_1.cs)]  
   
- Structs share most of the same syntax as classes, although structs are more limited than classes:  
```

Comment:
You might want to add here something like, " The name of the struct must be a valid C# \[identifier name](../inside-a-program/identifier-names.md)."